### ISSUE TITLE
Debugging .9 with php7.1?

### ISSUE THREAD

**Issue by ryannix123** [icewind1991/SMB#54]
Hello,

I recently built a new Nextcloud 11 server on a CentOS 7.3 system using Remi's 7.1 repo. Currently I have php71-php-smbclient.x86_64 (0.9.0-0.2.20161104git1857016.el7.remi) installed, however, I'm having some issues moving files through NextCloud to the SMB storage through webui's move command. I know the libphpsmb module is enabled, however, I'm getting some data corruption when larger files (>300MBs) are moved to the mounted SMB share. This doesn't happen with smaller text files. https://nextcloud.sesp.northwestern.edu/ncphp.php The mount is happening through [NextCloud's supported JSON config.](https://docs.nextcloud.com/server/9/admin_manual/configuration_files/external_storage/smb.html)

Basically, I uploaded a video file to Nextcloud, then attempted to move to the SMB mounted share through the browser. The file copied 90% of the data, and so I see it on the file share. However, I get an error that the data couldn't be moved so now I have a file in two places, Nextcloud and the file share. Not sure where the error is happening, or what needs to be tweaked. 

How can I debug this?

Thanks!

Ryan

**Comment by ryannix123** [comment]
No, our current ownCloud 9.1.2 instance with php 5.4 and php5-libsmbclient.x86_64 0.7.0-10.2oc             @isv_ownCloud_community_8.1 works just fine.

**Comment by ryannix123** [comment]
Tried another file. Here is the output from ownCloud's logs:

Uncaught Error: Access to undeclared static property: OC\Files\Filesystem::$normalizedPathCache in /home/nextcloud/public_html/lib/private/Files/Filesystem.php:806 Stack trace: #0 /home/nextcloud/public_html/lib/private/Files/View.php(2018): OC\Files\Filesystem::normalizePath('/rtn361/files/S...') #1 /home/nextcloud/public_html/lib/private/Files/View.php(1156): OC\Files\View->unlockFile('/SMB/Christmas ...', 1) #2 [internal function]: OC\Files\View->OC\Files\{closure}() #3 /home/nextcloud/public_html/apps/files_external/3rdparty/icewind/streams/src/CallbackWrapper.php(109): call_user_func(Object(Closure)) #4 [internal function]: Icewind\Streams\CallbackWrapper->stream_close() #5 {main} thrown at /home/nextcloud/public_html/lib/private/Files/Filesystem.php#806
